### PR TITLE
[WIP] Fix #2014 : Fill challenge's queue field with a random string

### LINKED ISSUE #2014: Update seed script to add broker url in challenge
## Current Behaviour:

Currently, when the challenge is created using the seed script, the `queue` field is not populated.

## Expected Behaviour:
When the seed script is run, the `queue` field must be populated with the random string of 75 characters in length.

### PR DESCRIPTION
**Pull request by yashdusing**
Fix #2014 : Fill challenge's queue field with a random string

### CODE REVIEW

**Comment by yashdusing** [comment]
@RishabhJain2018 Could you please verify now ?
Also I've left my proposal for review on your gitter could you please take a look at the proposal when you have the time to.

File: scripts/seed.py [hunk 1]
```diff
@@ -134,6 +136,7 @@ def create_challenge(title, start_date, end_date, host_team):
     """
     evaluation_script = open(
         os.path.join(settings.BASE_DIR, 'examples', 'example1', 'sample_evaluation_script.zip'), 'rb')
+    random_string = ''.join(random.choice(string.ascii_letters) for _ in xrange(75))
```
Comment (RishabhJain2018) [inline scripts/seed.py]: Please change it to `queue`

File: scripts/seed.py [hunk 2]
```diff
@@ -149,6 +152,7 @@ def create_challenge(title, start_date, end_date, host_team):
         anonymous_leaderboard=False,
         start_date=start_date,
         end_date=end_date,
+        queue=random_string,
```
Comment (RishabhJain2018) [inline scripts/seed.py]: `queue=queue`

File: scripts/seed.py (hunk 1 above)
Comment (RishabhJain2018) [inline scripts/seed.py]: Also, there is no `xrange` in Python3. Please change it to `range`.

Comment (yashdusing) [reply scripts/seed.py]: About that, at the start of the file, there’s xrange = range for python so this should work ever for python 3.

Comment (RishabhJain2018) [reply scripts/seed.py]: I see. But it would be nice if we follow only Python 3 syntax here. Can you please change it so that I can merge it.

**Comment by yashdusing** [comment]
@RishabhJain2018 Made the 'range' change

**Pull request merged**